app.R

```diff
@@ -1,50 +1,155 @@
-#
-# This is a Shiny web application. You can run the application by clicking
-# the 'Run App' button above.
-#
-# Find out more about building applications with Shiny here:
-#
-#    http://shiny.rstudio.com/
-#
-
 library(shiny)
+library(lubridate)
+library(leaflet)
+source("mydata.R")
 
-# Define UI for application that draws a histogram
-ui <- fluidPage(
-   
-   # Application title
-   titlePanel("Old Faithful Geyser Data"),
-   
-   # Sidebar with a slider input for number of bins 
-   sidebarLayout(
-      sidebarPanel(
-         sliderInput("bins",
-                     "Number of bins:",
-                     min = 1,
-                     max = 50,
-                     value = 30)
-      ),
-      
-      # Show a plot of the generated distribution
-      mainPanel(
-         plotOutput("distPlot")
-      )
-   )
-)
-
-# Define server logic required to draw a histogram
-server <- function(input, output) {
-   
-   output$distPlot <- renderPlot({
-      # generate bins based on input$bins from ui.R
-      x    <- faithful[, 2] 
-      bins <- seq(min(x), max(x), length.out = input$bins + 1)
+
+#get the first date on which a malaria case was recorded
+min_date <- read.csv("/home/mkhuphuli/hello/credentials.csv", header=TRUE) %>%
```

Comment:
How is the `credentials.csv` file structured?
1) If we're reading info from a file, we might want to store those once and re-use them so that we only read the file once
2) We want to try and make the code as re-usable as possible, so if I download and run this code, I would need to create an `mkhuphuli` dir or change the code.
3) There might also be a better way of storing the data with a JSON file

We've used something like this in our previous R files:
`json_data <- fromJSON(file=file.path(Sys.getenv("HOME"), "secrets.json"))`
and then to access items from the file:
`json_data$password`